monkestation/code/game/objects/items/mothlet_grenade.dm

```diff
@@ -0,0 +1,34 @@
+/obj/item/grenade/frag/mothlet //Grenade
+	name = "mothlet grenade"
+	desc = "CAUTION : DUBIOUS LITTLE CREATURES INSIDE."
+	icon = 'monkestation/icons/obj/mothletgrenade.dmi'
+	icon_state = "fragmoth"
+	shrapnel_type = /obj/projectile/bullet/shrapnel/mothlet
+	shrapnel_radius = 3
+	ex_heavy = 0 //The grenade flings the moths with c02 so no structural damage to anyone (or the moths)
+	ex_light = 0
+	ex_flame = 0
+
+/obj/projectile/bullet/shrapnel/mothlet //Projectile launched
+	name = "Soaring Mothlet"
```

Comment:
not a proper noun

Reply:
Soaring is a noun
The act of gliding while maintaining altitude, especially the sport of flying a heavier-than-air craft by using ascending currents of air.

Reply:
Specifically a _proper_ noun.

Jim is a proper noun, table is not